apps/web/utils/navigation.ts

```diff
@@ -12,9 +12,12 @@ interface Navigatable {
   title: string
 }
 
-const isNavigatable = (slice: any): slice is Navigatable => {
-  return typeof slice === 'object' && slice.id && slice.title
-}
+const isNavigatable = (slice: Slice) =>
+  typeof slice === 'object' &&
+  slice['id'] &&
+  slice['title'] &&
+  slice.__typename !== 'Image' &&
+  (slice['showTitle'] === undefined || slice['showTitle'])
```

Comment:
I find this clearer. Maybe add a comment on the intended behaviour.
```suggestion
  (slice['showTitle'] ?? true)
```

Reply:
I've added a comment and changed the line